### DISCUSSION TITLE
付款后还是没有升级到专业版

### DISCUSSION THREAD

**Discussion by xy777981** [getlantern/lantern-forum-cn#796] [Q&A]
邮箱：cnk2009@yeah.net
![微信图片_20240819194734](https://github.com/user-attachments/assets/b3a00a97-37ab-4982-9b93-997a8ca27037)

**Comment by MaxMKMax** [comment]
@xy777981 您好. 已经帮您提交问题上去了. 谢谢您支持蓝灯!
